src/test/java/com/amplitude/HttpTransportTest.java

```diff
@@ -95,8 +95,10 @@ public void onLogEventServerResponse(Event event, int status, String message) {
     assertTrue(latch.await(1L, TimeUnit.SECONDS));
     verify(httpCall, times(4)).makeRequest(anyList());
     for (int i = 0; i < events.size(); i++) {
-      if (i < (events.size() / 2)) {
+      if (i < (events.size() / 4)) {
         assertEquals(200, resultMap.get(events.get(i)));
+      } else if (i < (events.size() / 2)) {
+        assertEquals(413, resultMap.get(events.get(i)));
```

Comment:
The reason here is for add reduce for 413?

Reply:
Yes, since I add reduce payload to operation for status 413. The first rate limit response cut and drop half of the list makes the last 5 callbacks have code 429. The second payload too large response cut and drop 3 events out of 5.
The rest 2 events get success response and callback code is 200.